proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,239 @@
+# dotnet nuget trust command
+
+- Author Name [Kat Marchan (@zkat)](https://github.com/zkat)
+- Start Date 2021-03-04
+- GitHub Issue: https://github.com/NuGet/Home/issues/10634
+- GitHub PR: https://github.com/NuGet/Home/pull/10628
+
+## Summary
+
+This specifies the syntax and behavior for the `dotnet nuget trust` command.
+
+## Motivation
+
+This is part of a larger effort to reach full parity between `nuget` and
+`dotnet` for NuGet-related operations.
+
+## Explanation
+
+### Functional explanation
+
+#### `> dotnet nuget trust`
+
+```
+Provides the ability to manage the list of trusted signers.
+
+USAGE:
+    dotnet nuget trust [COMMAND] [OPTIONS]
+
+COMMANDS:
+    list
+    author <name> <package>...
+    repository <name> <package>...
+    source <name> [source-url]
+    certificate <name> <fingerprint>
+    remove <name>
+    sync <name>
+```
+
+#### `> dotnet nuget trust list`
+
+```
+Lists all the trusted signers in the configuration. This option will include all the certificates (with fingerprint and fingerprint algorithm) each signer has. If a certificate has a preceding [U], it means that certificate entry has allowUntrustedRoot set as true.
```

Comment:
What do you think of this?

```suggestion
Lists all the trusted signers in the configuration. This option will include all the certificates (with fingerprint and fingerprint algorithm) each signer has. If a certificate has a preceding [U], NuGet will not reject the certificate if it chains to an untrusted root.
```

Reply:
I'm not sure. I think it's good to make it clear that the `[U]` corresponds to an actual option in the config file, you know? What do you think, @heng-liu ?

Reply:
Hm... IMHO ideally no one ever uses `allowUntrustedRoot`. If so, maybe we shouldn't bring up such an advanced topic in the command description? It is difficult to balance between useful information and verbosity. What do you think of only explaning `[U]` in command output if the user has `allowUntrustedRoot` enabled somewhere as per https://github.com/NuGet/Home/pull/10628#discussion_r592009496? If so, we can detail both the `allowUntrustedRoot` config plus give a quick human description of what that entails.